components/eam/src/physics/cam/flux_avg.F90

```diff
@@ -86,37 +82,31 @@ subroutine flux_avg_init(cam_in,  pbuf2d)
       call pbuf_set_field(pbuf2d_chunk, taux_idx,   cam_in(lchnk)%wsx(:ncol))
       call pbuf_set_field(pbuf2d_chunk, tauy_idx,   cam_in(lchnk)%wsy(:ncol))
 
-      call pbuf_set_field(pbuf2d,       shflx_res_idx, 0.0_r8)
+      call pbuf_set_field(pbuf2d_chunk, shflx_res_idx, 0.0_r8)
       call pbuf_set_field(pbuf2d_chunk, lhflx_res_idx, 0.0_r8)
       call pbuf_set_field(pbuf2d_chunk, qflx_res_idx,  0.0_r8)
       call pbuf_set_field(pbuf2d_chunk, taux_res_idx,  0.0_r8)
       call pbuf_set_field(pbuf2d_chunk, tauy_res_idx,  0.0_r8)
    end do
 
-
 end subroutine flux_avg_init
 
 !===============================================================================
 
 subroutine flux_avg_run(state, cam_in,  pbuf, nstep, deltat)
   use physics_buffer, only : physics_buffer_desc, pbuf_get_field
-   !----------------------------------------------------------------------- 
-   ! 
-   ! Purpose: 
-   !
-   !----------------------------------------------------------------------- 
-!++ debug code to be removed after PBL code validated
-   use phys_debug,       only: phys_debug_flux1, phys_debug_flux2
-!-- debug code to be removed after PBL code validated
-
+   !---------------------------------------------------------------------------- 
+   ! Purpose: smooth fluxes from surface components
+   !----------------------------------------------------------------------------
    ! Input arguments
-
    type(physics_state), intent(in)    :: state
    type(cam_in_t),      intent(inout) :: cam_in
    type(physics_buffer_desc), pointer :: pbuf(:)
    
    integer,             intent(in)    :: nstep
    real(r8),            intent(in)    :: deltat
+   
+   real(r8), parameter :: timescale = 7200 ! smoothing time scale [s]
```

Comment:
halving the timescale here will also affect the behavior of non-MMF. I don't see no harm reducing the smoothing time scale if it is non-MMF or MMF. I am just curious does non-MMF also use flux_avg method by default?

Reply:
non-MMF E3SM configurations do not use the surface flux smoothing, so this wont affect anyone but us. And the 2-hour time scale was in the original version, which I had made longer because I thought it might help with the early MMF instability problems, so this restores the original behavior of this option. Not that it really matters. I thought about making this a namelist parameter, but I dont think it's worth it.